machines/Issuers/IssuersMachine.ts

```diff
@@ -326,31 +326,30 @@ export const IssuersMachine = model.createMachine(
           src: 'verifyCredential',
           onDone: [
             {
-              actions: ['sendSuccessEndEvent'],
+              actions: ['sendSuccessEndEvent', 'setIsVerified'],
               target: 'storing',
             },
           ],
           onError: [
+            {
+              cond: 'isPendingVerificationError',
+              actions: ['resetLoadingReason', 'resetIsVerified'],
+              target: 'storing',
+            },
             {
               actions: [
-                log('Verification Error.'),
                 'resetLoadingReason',
-                'updateVerificationErrorMessage',
                 'sendErrorEndEvent',
+                'setErrorAsVerificationError',
               ],
-              //TODO: Move to state according to the required flow when verification of VC fails
               target: 'handleVCVerificationFailure',
             },
           ],
         },
       },
 
       handleVCVerificationFailure: {
-        on: {
-          RESET_VERIFY_ERROR: {
-            actions: ['resetVerificationErrorMessage'],
-          },
-        },
+        entry: 'sendVerificationError',
```

Comment:
Why are we removing this event?
RESET_VERIFY_ERROR -> this event is required to dismiss the error screen, right?